stix2slider/convert_cyber_observables.py

```diff
@@ -179,10 +198,11 @@ def convert_autonomous_system_c_o(as20, as1x, obs20_id):
     convert_obj(as20, as1x, AUTONOMOUS_SYSTEM_MAP, obs20_id)
 
 
-def convert_domain_name_c_o(dn20, dn1x):
+def convert_domain_name_c_o(dn20, dn1x, obs20_id):
```

Comment:
At some point I'd like to rethink our logging approach. It seems awkward to have to pass in a parameter just for use in (warning) log messages.

Reply:
ideas?

Reply:
Not an issue for this PR, but maybe create a special Logger subclass, where you could set a "context" attribute (the ID of the part of the STIX 2 JSON currently being processed), which could be automatically incorporated into the log message? I'll look into an approach like this.